lib/nats.js

```diff
@@ -17,7 +17,7 @@ var net    = require('net'),
     url    = require('url'),
     util   = require('util'),
     events = require('events'),
-    nuid   = require('./nuid.js');
+    nuid   = require('nuid');
```

Comment:
I got freaked out, that I had left a trailed ref to ./nuid.js, but that is
my change list :)
Yes it got pushed out and it is available already.

/a



On August 11, 2016 at 2:08:55 PM, Alberto Ricart (alberto.ricart@apcera.com)
wrote:

Yes we did. Let me fix that.

/a



On August 11, 2016 at 2:07:58 PM, Derek Collison (notifications@github.com)
wrote:

In lib/nats.js
<https://github.com/nats-io/node-nats/pull/86#discussion_r74483979>:

> @@ -17,7 +17,7 @@ var net    = require('net'),
>      url    = require('url'),
>      util   = require('util'),
>      events = require('events'),
> -    nuid   = require('./nuid.js');
> +    nuid   = require('nuid');

So we pushed nuid to npm?

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/nats-io/node-nats/pull/86/files/0eed8b8195c911f7e0d4410d63718da804604cdd#r74483979>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AA_DEK5UUx48V_AB5urUMvf-FXP-2ys7ks5qe3MOgaJpZM4JiVzm>
.